index.html

```diff
@@ -34,14 +35,17 @@ <h1 class="header">John Doe</h1>
                 <a href="https://www.linkedin.com/in/tetteh-kodjo-sarso-888989125/" class="social-link">
                     <img src="assets/images/linkedin.png" alt="" class="icon">
                 </a>
+                <a href="#" class="social-link">
+                    <img src="assets/images/angelist 1.png" alt="" class="icon">
+                </a>
                 
                 <a href="https://twitter.com/tksarso" class="social-link">
                     <img src="assets/images/twitterIcon.png" alt="" class="icon">
                 </a>
+                <a href="#" class="social-link">
+                    <img src="assets/images/Vector.png" alt="" class="icon">
+                </a>
```

Comment:
- [Optional] I think the list of social icons should be an unordered list. This is best for HTML semantics. The list should look like 
```html
<ul>
  <li>
    <a><img></a>
  </li>
  <li>
    <a><img></a>
  </li>
</ul>
```

Reply:
ohh okay.. let me fix that right now..